odl/operator/default_ops.py

```diff
@@ -754,15 +756,17 @@ class ConstantOperator(Operator):
         ``ConstantOperator(y)(x) == y``
     """
 
-    def __init__(self, vector, domain=None):
+    def __init__(self, vector, domain=None, range=None):
         """Initialize a new instance.
 
         Parameters
         ----------
         vector : `LinearSpaceElement`
             The constant space element to be returned.
         domain : `LinearSpace`, optional
-            Domain of the operator. Default : ``vector.space``.
+            Domain of the operator. Default: ``vector.space``
+        range : `LinearSpace`, optional
```

Comment:
Also the doc of `constant` needs to be updated since we cast if `range` is given. Suggestion:
```
constant: `LinearSpaceElement` or ``range`` element-like
    The constant space element to be returned. If ``range`` is not 
    provided, ``constant`` must be a `LinearSpaceElement` since the
    operator range is then inferred from it.
```
Possibly with single backticks around "element-like" since we have a glossary term for it. Could however clutter the row, we'll have to look at it.